HealthyTests/Classes/Modules/SavedRecipes/Cell/SavedRecipesTableViewCellTests.swift

```diff
@@ -0,0 +1,53 @@
+import XCTest
+@testable import Healthy
+
+final class SavedRecipesTableViewCellTests: XCTestCase {
+
+    // MARK: Properties
+    
+    private var sut: SavedRecipesTableViewCell!
+    private var mockTableView: SavedRecipesTableViewMock!
+    
+    // MARK: Lifecycle
+    
+    override func setUp() {
+        sut = SavedRecipesTableViewCell()
+        mockTableView = SavedRecipesTableViewMock(frame: CGRect(x: 0, y: 0, width: 200, height: 400), style: .plain)
+        mockTableView.register(UINib.init(nibName: "SavedRecipesTableViewCell", bundle: nil),
```

Comment:
Please check how we register the collection view cells and implement a similar approach. Feel free to alugn with @abdallah-7698